Core/Strings/Default.axaml

```diff
@@ -11,7 +11,8 @@
 	<!-- ApplicationCulture -->
 	<sys:String x:Key="String/ApplicationCulture.EN_US">English (United States)</sys:String>
 	<sys:String x:Key="String/ApplicationCulture.System">System</sys:String>
-	<sys:String x:Key="String/ApplicationCulture.ZH_TW">正體中文 (台灣)</sys:String>
+	<sys:String x:Key="String/ApplicationCulture.ZH_TW">正體中文 (中国台灣)</sys:String>
```

Comment:
Do not change this string for political reason.

Reply:
Sorry, I wan't involve about political, I just think this is the truth and provider the specification.

Reply:
Whether it is truth or not is a political issue. You can change ```String/ApplicationCulture.ZH_TW``` in ```zh-CN.axaml``` instead of changing in ```Default.axaml```.